reference/sql-commands/copy-to.md

```diff
@@ -150,3 +159,28 @@ CREATE CREDENTIALS my_aws_creds PROVIDER aws OPTIONS (
 COPY public.users TO 's3://bucket/users.json' FORMAT json
 CREDENTIALS my_aws_creds ( region '<aws_region>' );
 ```
+
+### Copying files to the Delta table format
+
+Queries can be copied to Delta tables by passing in an absolute path to a
+blank directory (which will become the Delta table).
```

Comment:
```suggestion
empty directory, which will become the Delta table.
```

Reply:
be clear that the directory must exist and not have any delta tables in it. if people want to append to an existing table, use insert into.